src/components/Footer/Footer.js

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```

Comment:
Isn't any other way to do this without adding a new library?

Reply:
this library we current use now.. why new library?

Reply:
If I am not wrong, it is not a new library, it is another helper method from the same library. @fcoronelmakingsense ?

Reply:
I got confused by the pr description, my mistake.

Reply:
Anyway, it looks a little complicated, it smells like our solution is not the best. Maybe there is a different alternative than translate a property.